tedge/src/cli/disconnect/mod.rs

```diff
@@ -1,159 +1,3 @@
-        // Check if bridge exists and stop with code 0 if it doesn't.
-        let bridge_conf_path =
-            paths::build_path_for_sudo_or_user(&[TEDGE_BRIDGE_CONF_DIR_PATH, &self.config_file])?;
-
-        println!("Removing {} bridge.\n", self.cloud_name);
-        match std::fs::remove_file(&bridge_conf_path) {
-            // If we find the bridge config file we remove it
-            // and carry on to see if we need to restart mosquitto.
-            Ok(()) => Ok(()),
-
-            // If bridge config file was not found we assume that the bridge doesn't exist,
-            // We finish early returning exit code 0.
-            Err(e) if e.kind() == std::io::ErrorKind::NotFound => {
-                println!("Bridge doesn't exist. Operation finished!");
-                Err(DisconnectBridgeError::BridgeFileDoesNotExist)
-            }
-
-            Err(e) => Err(DisconnectBridgeError::FileOperationFailed(
-                e,
-                bridge_conf_path,
-            )),
-        }
-    }
-
-    // Deviation from specification:
-    // Check if mosquitto is running, restart only if it was active before, if not don't do anything.
-    fn apply_changes_to_mosquitto(
-        &self,
-        user_manager: &UserManager,
-    ) -> Result<(), DisconnectBridgeError> {
-        println!("Applying changes to mosquitto.\n");
-        if MosquittoService.is_active()? {
-            MosquittoService.restart(user_manager)?;
-            println!("{} Bridge successfully disconnected!\n", self.cloud_name);
-        }
-        Ok(())
-    }
-
-    fn stop_and_disable_tedge_mapper(&self, user_manager: &UserManager) {
-        let _root_guard = user_manager.become_user(ROOT_USER);
-        let mut failed = false;
-
-        println!("Stopping tedge-mapper service.\n");
-        if let Err(err) = TedgeMapperService.stop(user_manager) {
-            println!("Failed to stop tedge-mapper service: {:?}", err);
-            failed = true;
-        }
-
-        println!("Disabling tedge-mapper service.\n");
-        if let Err(err) = TedgeMapperService.disable(user_manager) {
-            println!("Failed to disable tedge-mapper service: {:?}", err);
-            failed = true;
-        }
-
-        if !failed {
-            println!("tedge-mapper service successfully stopped and disabled!\n");
-        }
-    }
-}
-
-#[derive(thiserror::Error, Debug)]
-pub enum DisconnectBridgeError {
-    #[error(transparent)]
-    Configuration(#[from] ConfigError),
-
-    #[error("File operation error. Check permissions for {1}.")]
-    FileOperationFailed(#[source] std::io::Error, String),
-
-    #[error(transparent)]
-    IoError(#[from] std::io::Error),
-
-    #[error(transparent)]
-    PathsError(#[from] paths::PathsError),
-
-    #[error(transparent)]
-    ServicesError(#[from] services::ServicesError),
-
-    #[error("Bridge file does not exist.")]
-    BridgeFileDoesNotExist,
-}
+pub mod cli;
```

Comment:
It's minor. If you add this here,
```suggestion
pub use self::cli::TEdgeDisconnectBridgeCli;
```
you can remove `cli::` from `tedge/src/cli/mod.rs`. 
```
 Disconnect(disconnect::TEdgeDisconnectBridgeCli),
```
The code looks more consistent with other sub-commands, and this is what we did in my PR #182.  At least, we should decide either your way or my way. Mixing them up in the end is most ugly :P

Reply:
Sure, works for me.